charts/kommander-operator/values.yaml

```diff
@@ -0,0 +1,13 @@
+# Default values for kommander-operator
+# this is using a nginx image at the moment that will be replaced by proper image
+
+replicaCount: 1
+image:
+  repository: nginx
+  tag: 0.1.0
+  pullPolicy: IfNotPresent
+
+kubetools:
+  image:
+    repository: mesosphere/kommander2-kubetools
+    tag: v2.4.0
```

Comment:
Could you add a note (and maybe a ticket in jira) so we don't forget to bind it with the kommander version?